apps/box/src/screens/Settings/ComponentGallery/TableDemo.screen.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  FxSafeAreaBox,
+  FxSpacer,
+  FxTable,
+} from '@functionland/component-library';
+import { HeaderText, SubHeaderText } from '../../../components/Text';
+import React from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
+
+const ICON_BUFFER = 32;
+
+export const TableDemoScreen = () => {
+  return (
+    <FxSafeAreaBox flex={1}>
+      <ScrollView>
+        <HeaderText>Table Demo</HeaderText>
+        <FxSpacer marginTop="16" />
+        <SubHeaderText>Without foldout</SubHeaderText>
+        <FxTable>
+          <FxTable.Header>
+            <FxTable.Title>First</FxTable.Title>
+            <FxTable.Title>Second</FxTable.Title>
+            <FxTable.Title>Third</FxTable.Title>
+          </FxTable.Header>
+
+          <FxTable.Row>
+            <FxTable.Cell variant="bodyLargeRegular">Test 1</FxTable.Cell>
+            <FxTable.Cell>Test 1</FxTable.Cell>
+            <FxTable.Cell>Test 1</FxTable.Cell>
+          </FxTable.Row>
+          <FxTable.Row>
+            <FxTable.Cell variant="bodyLargeRegular">Test 2</FxTable.Cell>
+            <FxTable.Cell>Test 2</FxTable.Cell>
+            <FxTable.Cell>Test 2</FxTable.Cell>
+          </FxTable.Row>
+          <FxTable.Row>
+            <FxTable.Cell variant="bodyLargeRegular">Test 3</FxTable.Cell>
+            <FxTable.Cell>Test 3</FxTable.Cell>
+            <FxTable.Cell>Test 3</FxTable.Cell>
+          </FxTable.Row>
+        </FxTable>
+
+        <FxSpacer marginTop="16" />
+        <SubHeaderText>With foldout</SubHeaderText>
+        <FxTable>
+          <FxTable.Header>
+            <FxTable.Title width={ICON_BUFFER} />
```

Comment:
`width` should be less the header padding in this case `12`.

Reply:
Doing this messes up the alignment with the header items and row contents.